common/scripts/ci/run-sdk-backend-tiger-integration-tests-with-live-backend.sh

```diff
@@ -6,7 +6,7 @@ ROOT_DIR=$(echo $(cd $(dirname "${BASH_SOURCE[0]}")/../../../ && pwd -P))
 _RUSH="${DIR}/docker_rush.sh"
 _RUSHX="${DIR}/docker_rushx.sh"
 
-NODE_IMAGE='020413372491.dkr.ecr.us-east-1.amazonaws.com/tools/gdc-frontend-node-18:node-18.17.0-yarn-1.22.19'
+NODE_IMAGE='020413372491.dkr.ecr.us-east-1.amazonaws.com/3rdparty/library/node:18'
```

Comment:
is there a special reason for this?, if not please keep it as it is for this scope of change